.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ orbs:
 jobs:
   build:
     docker:
-      - image: 'citusdata/extbuilder:werror'
+      - image: 'heisenberg302/build:latest'
```

Comment:
please use the images in either `citusdata` or `citus`

Reply:
I will update them once we agree on the other details because I want to put the final image to `citus`.